arch/arm64/boot/dts/rockchip/rk3566-firefly-roc-pc.dts

```diff
@@ -0,0 +1,1605 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Copyright (c) 2020 Rockchip Electronics Co., Ltd.
+ *
+ */
+
+ /dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pinctrl/rockchip.h>
+#include <dt-bindings/display/rockchip_vop.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pwm/pwm.h>
+#include <dt-bindings/pinctrl/rockchip.h>
+#include <dt-bindings/input/rk-input.h>
+#include <dt-bindings/display/drm_mipi_dsi.h>
+#include <dt-bindings/sensor-dev.h>
+#include "rk3566.dtsi"
+#include "rk3568-linux.dtsi"
+
+#define M2_SATA_OR_PCIE 0 /* 1 = SATA, 0 = PCIe */
```

Comment:
We don't need this one. We aldready have sata overlays

Reply:
OK, thank you for your reminder.